integration/white_box_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
+	"github.com/sirupsen/logrus"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/portforward"
+)
+
+func TestPortForward(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	ns, _, deleteNs := SetupNamespace(t)
+	defer deleteNs()
+	dir := "examples/microservices"
+	skaffold.Run().InDir(dir).InNs(ns.Name).RunOrFailOutput(t)
+	defer skaffold.Delete().InDir(dir).InNs(ns.Name).RunOrFailOutput(t)
```

Comment:
You can remove the delete. I’ll be removed with the ns

Reply:
right, that makes sense!